.buildkite/pull-requests.json

```diff
@@ -15,6 +15,22 @@
             "skip_target_branches": [ ],
             "skip_ci_on_only_changed": [ ],
             "always_require_ci_on_changed": [ ]
+        },
+        {
+            "enabled": true,
+            "pipelineSlug": "beats",
+            "allow_org_users": true,
+            "allowed_repo_permissions": ["admin", "write"],
+            "allowed_list": [ ],
+            "set_commit_status": true,
+            "build_on_commit": true,
+            "build_on_comment": true,
+            "trigger_comment_regex": "^/test metricbeat$",
+            "always_trigger_comment_regex": "^/test metricbeat$",
+            "skip_ci_labels": [  ],
+            "skip_target_branches": [ ],
+            "skip_ci_on_only_changed": [ ],
+            "always_require_ci_on_changed": [ ]
```

Comment:
**question**: will this field be extended with the list of files that will always trigger the build?

```suggestion
            "always_require_ci_on_changed": [ "^filebeat/.*", "^x-pack/filebeat/.*", ]
```

[x-pack/filebeat](https://github.com/elastic/beats/blob/ead3090552292e39adb56e2a6e069b0787bbe83c/x-pack/filebeat/Jenkinsfile.yml#L3-L6) and [filebeat](https://github.com/elastic/beats/blob/main/filebeat/Jenkinsfile.yml#L3-L6) are the current list of files that will trigger a build. And those aliases can be found in [here](https://github.com/elastic/beats/blob/ead3090552292e39adb56e2a6e069b0787bbe83c/Jenkinsfile.yml#L21-L38)